mod_form.php

```diff
@@ -223,4 +237,18 @@ public function completion_rule_enabled($data) {
         return !empty($data['completionsubmit']);
     }
 
+    /**
+     * Create options for remove old responses in the questionare.
+     *
+     * @return array
+     */
+    public function create_remove_options() {
```

Comment:
This function doesn't appear to be used anywhere. It may be a duplicate of locallib::questionnaire_create_remove_options()? If so, it may make sense to use this one rather than the one in locallib?